guides/common/modules/proc_cloning_satellite_server.adoc

```diff
@@ -166,7 +147,6 @@ You can either mount the shared storage or copy the backup files to the `/backup
 [options="nowrap" subs="quotes,attributes"]
 ----
 # subscription-manager register _your_customer_portal_credentials_
-# subscription-manager attach --pool=__pool_ID__
```

Comment:
See also "attach subscriptions" on line 164.

Reply:
Thanks for spotting that!